packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -428,19 +428,19 @@ export class ExecutionEngineHttp implements IExecutionEngine {
 
     switch (newState) {
       case ExecutionEngineState.ONLINE:
-        this.logger.info("ExecutionEngine became online");
+        this.logger.info("Execution client became online");
         break;
       case ExecutionEngineState.OFFLINE:
-        this.logger.error("ExecutionEngine went offline");
+        this.logger.error("Execution client went offline");
         break;
       case ExecutionEngineState.SYNCED:
-        this.logger.info("ExecutionEngine is synced");
+        this.logger.info("Execution client is synced");
         break;
       case ExecutionEngineState.SYNCING:
-        this.logger.info("ExecutionEngine is syncing");
+        this.logger.warn("Execution client is syncing");
```

Comment:
I think this should not be a warning. It's a normal scenario that will happen a lot of times and syncing is a normal process not something a user should be worried about.

Reply:
My assumption here is that this is only printed out if the state changes, therefore this should not happen often during normal operation, if the EL gets into syncing mode there is degraded performance, the node might not be able to perform correct head votes / miss block proposals.

Reply:
i think warn level is good here because it will only come if execution goes into syncing from synced, which is actually not supposed to happen often once execution syncs up with the CL

Reply:
I think it's the sync stats among the EL nodes, not the with the CL. And EL nodes can get unsynced and synced a lot?

Reply:
> sync stats among the EL nodes

That's an interesting topic, how does this actually work if you specify multiple ELs in `--execution.urls`.

I see several issues with this
- if one beacon node has multiple ELs only one gets payload and fork choice updates, how are the fallback ELs able to sync? I thought this requires some sort of multiplexer / middleware that sends request to all ELs.
- if a execution client is connected to multiple beacon nodes it causes frequent reorgs as it might received different fork choice updates

Reply:
> EL nodes can get unsynced and synced a lot

At least in the normal case with just one execution client I have not yet seen that the EL is getting back to syncing mode

Reply:
> I think it's the sync stats among the EL nodes, not the with the CL. And EL nodes can get unsynced and synced a lot?

no they can't once synced they should stay synced with the CL head as long as CL doesn't go out of sync and then CL doesn't has to do batch updates to EL (our req/resp is batch processing) to get back to head

Reply:
I think warn should be better then, let's keep the change